src/SFA.DAS.ApprenticeAccounts.UnitTests/Application/Queries/MyApprenticeship/WhenHandlingGetMyApprenticeship.cs

```diff
@@ -0,0 +1,90 @@
+﻿using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApprenticeAccounts.Application.Commands.CreateMyApprenticeCommand;
+using SFA.DAS.ApprenticeAccounts.Application.Queries.MyApprenticeshipQuery;
+using SFA.DAS.ApprenticeAccounts.Data;
+using SFA.DAS.ApprenticeAccounts.Data.Models;
+using SFA.DAS.ApprenticeAccounts.DTOs.MyApprenticeship;
+using SFA.DAS.Testing.AutoFixture;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mail;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeAccounts.UnitTests.Application.Queries.MyApprenticeship;
+
+public class WhenHandlingGetMyApprenticeship
+{
+    [Test]
+    [RecursiveMoqAutoData]
+    public async Task ThenExpectedMyApprenticeshipsAreReturned(
+            MyApprenticeshipQuery query,
+            Mock<IApprenticeContext> mockApprenticeContext,
+            Mock<IMyApprenticeshipContext> mockMyApprenticeshipContext,
+            ApprenticeWithMyApprenticeshipsDto apprenticeWithMyApprenticeships,
+            Guid apprenticeId
+        )
+    {
+        var myApprenticeships = new List<Data.Models.MyApprenticeship>();
+        foreach (var apprenticeship in apprenticeWithMyApprenticeships.MyApprenticeships)
+        {
+            apprenticeship.Id = apprenticeId;
+            var command = new CreateMyApprenticeshipCommand
```

Comment:
This should be `MyApprenticeship` and not `CreateMyApprenticeshipCommand`.
I was wondering why is a context returning a type that has command like name  `CreateMyApprenticeshipCommand`? 🤯 I had to dig in 673 to see the interface and operator.

Reply:
I'll change it to MyApprenticeship, but this was done to  keep consistency with the other "commands" in this repo eg ChangeEmailAddressCommand, UpdateApprenticeCommand